### ISSUE TITLE
error occur on folia

### ISSUE THREAD

**Issue by cgw2023** [iRedTea/NoDropX#1]
folia 1.20.2 
NoDropX v1.0.3c-SNAPSHOT

when drop item, error occur 
```
[22:34:58 ERROR]: Could not pass event PlayerDropItemEvent to NoDropX v1.0.3c-SNAPSHOT
me.redtea.nodropx.libs.tr7zw.nbt.NbtApiException: [?]Error while calling the method 'contains(java.lang.String)', loaded: true, Enum: COMPOUND_HAS_KEY, Passed Class: class java.lang.Boolean
        at me.redtea.nodropx.libs.tr7zw.nbt.utils.nmsmappings.ReflectionMethod.run(ReflectionMethod.java:358) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.libs.tr7zw.nbt.NBTReflectionUtil.getData(NBTReflectionUtil.java:635) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.libs.tr7zw.nbt.NBTCompound.hasTag(NBTCompound.java:624) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.service.nbt.impl.Tr7zwNBTServiceImpl.getNoDropTag(Tr7zwNBTServiceImpl.java:20) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.service.nodrop.impl.NoDropServiceImpl.isNoDrop(NoDropServiceImpl.java:23) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.service.event.impl.EventServiceImpl.callThrownEvent(EventServiceImpl.java:20) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at me.redtea.nodropx.listener.PlayerHandler.onDrop(PlayerHandler.java:31) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor266.execute(Unknown Source) ~[?:?]        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[folia-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[folia-api-1.20.2-R0.1-SNAPSHOT.jar:git-Folia-"3843340"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[folia-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[folia-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.world.entity.player.Player.drop(Player.java:778) ~[?:?]
        at net.minecraft.world.entity.player.Player.drop(Player.java:725) ~[?:?]
        at net.minecraft.server.level.ServerPlayer.drop(ServerPlayer.java:2786) ~[?:?]
        at net.minecraft.server.level.ServerPlayer.drop(ServerPlayer.java:2877) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handlePlayerAction(ServerGamePacketListenerImpl.java:1842) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundPlayerActionPacket.handle(ServerboundPlayerActionPacket.java:42) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundPlayerActionPacket.a(ServerboundPlayerActionPacket.java:15) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:55) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$2(PacketUtils.java:88) ~[?:?]
        at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:173) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1516) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:402) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:404) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:525) ~[folia-1.20.2.jar:git-Folia-"3843340"]
        at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalArgumentException: object is not an instance of declaring class
        at jdk.internal.reflect.DirectMethodHandleAccessor.checkReceiver(DirectMethodHandleAccessor.java:197) ~[?:?]
        at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:99) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
        at me.redtea.nodropx.libs.tr7zw.nbt.utils.nmsmappings.ReflectionMethod.run(ReflectionMethod.java:355) ~[NoDropX-1.0.3c-SNAPSHOT-all.jar:?]
        ... 28 more
>
```

**Comment by iRedTea** [comment]
I will fix it in 1.0.4

**Comment by iRedTea** [comment]
it was a problem of de.tr7zw:item-nbt-api library, but with the new version, they announced their support for Folia. Since NoDropX version 1.0.4-SNAPSHOT, a Folia-enabled version of this library has been included in the build. I think, the bug should be fixed
